app/Providers/TelescopeServiceProvider.php

```diff
@@ -63,10 +63,8 @@ protected function hideSensitiveRequestDetails()
     protected function gate()
     {
         Gate::define('viewTelescope', function ($user) {
-            return in_array($user->email, [
-                'psw58@cornell.edu',
-                'elw234@cornell.edu',
-            ]);
+            $cd_admins = explode(',', env('CD_ADMINS'));
```

Comment:
Just allows a little more robustness in the ENV file...
```suggestion
            $cd_admins = preg_split('/[, ]+/', env('CD_ADMINS'));
```